release/tar/linux/opensearch-dashboards-tar-build.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+
+#
+# About:        This shell script generate an opensearch-dashboards tarball with all the provided plugins installed and ready to use.
+#               It will create a folder named target that will consist of the tarball and shasum of the same.
+#               A manifest file and right permissions set up is essential to download the required artifacts
+# Dependencies: yq (More info: https://github.com/mikefarah/yq/tree/v4.4.1#install)
+#               shasum
+# Usage:        ./opensearch-dashboards-tar-build.sh -f <path/to/manifest/file/>
+#
+
+set -e
+
+function usage() {
+    echo "Usage: $0 [args]"
+    echo -e "Supported manifest schema-version: 1.0"
+    echo ""
+    echo "Arguments:"
+    echo -e "-f MANIFEST_FILE\t[Required] Path/to/manifest.yml file"
+    echo -e "-h help"
+}
+
+while getopts ":ha:p:f:" arg; do
```

Comment:
What is `a:` for?

Reply:
It was for arch. Will remove a and p as well. Thanks for catching

Reply:
Done